dart_io_mini_samples/example/files_directories_and_symlinks/symlinks/checking_if_a_path_represents_a_symlink.dart

```diff
@@ -2,17 +2,17 @@
 /// a symlink.
 
 import 'dart:io';
+import 'dart:async'; // Import not needed but added here to explicitly assign type for clarity below.
 
-void main() {
+main() async {
   // Get the system temp directory.
   var systemTempDir = Directory.systemTemp;
 
   // List the contents of the system temp directory.
-  systemTempDir.list(recursive: true, followLinks: false)
-    .listen((FileSystemEntity entity) {
-      // Print the path only if it represents a symlink.
-      FileSystemEntity.isLink(entity.path).then((isLink) {
-        if (isLink) print(entity.path);
-      });
-    });
+  Stream<FileSystemEntity> entityList = systemTempDir.list(recursive: true, followLinks: false); 
+  await for (FileSystemEntity entity in entityList) {
+  	// Print the path only if it represents a symlink.
```

Comment:
This looks like it's indented too far to me. (Maybe you used tabs instead of spaces?)